aspnetcore/performance/caching/distributed.md

```diff
@@ -39,6 +39,8 @@ Add a package reference for the distributed cache provider used:
 
 * For a Redis distributed cache, [Microsoft.Extensions.Caching.StackExchangeRedis](https://www.nuget.org/packages/Microsoft.Extensions.Caching.StackExchangeRedis).
 * For SQL Server, [Microsoft.Extensions.Caching.SqlServer](https://www.nuget.org/packages/Microsoft.Extensions.Caching.SqlServer).
+* For Postgres, [Microsoft.Extensions.Caching.Postgres](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Postgres).
```

Comment:
```suggestion
* For PostgreSQL, [Microsoft.Extensions.Caching.Postgres](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Postgres).
```
Should this be the full name PostgreSQL?  If PostgreSQL is the correct full name it would be best to go with that to help with translation and avoid consusion over which Postgres products or standards are being referred to.